### ISSUE TITLE
Projectile tragectory rotates unexpectedly when trying to boost them straight forward on ValkyrienSkies2 ships

### ISSUE THREAD

**Issue by 19PHOBOSS98** [LubieKakao1212/OpenCombatUtility#5]
Here I've setup a spell turret (Ars Nouveau) to fire projectiles over a pulsing OpenCU repulsor. I have hitboxes enabled for better observation. The whole setup is built on a ValkyrienSkies2 ship. Keep in mind the videos are spedup because of cmdcam+minema2 mod recording setup:
```
p=peripheral.find("opencu:repulsor")
p.recalibrateByIdx(1)
p.setRadius(3)
p.setForce(1)
p.setVector(1,0,0)

while(true) do
p.pulse(0,0,0)
sleep(0.1)
end

```
https://youtu.be/nccZDytK_2U

OpenCU version 1.3.2.2

EDIT:
It seems it works as expected in version 1.3.1

**Comment by LubieKakao1212** [comment]
Does it also happen while not on a VS2 ship?

**Comment by LubieKakao1212** [comment]
I think I have a fix, for both

**Comment by LubieKakao1212** [comment]
fixed in 1.3.2.3

**Comment by 19PHOBOSS98** [comment]
slr, thank you!
